src/test/ui/type-check/assignment-in-if.stderr

```diff
@@ -16,7 +16,7 @@ error[E0308]: mismatched types
 31 |     if (x = x) {
    |        ^^^^^^^
    |        |
-   |        help: did you mean to compare equality? `x == x`
+   |        help: did you mean to compare equality?: `x == x`
```

Comment:
Hm... those came in after a rebase. I'll create a new PR fixing up all suggestions that came in after I wrote the guidelines into the docs.